test/transactionBreakdown.test.ts

```diff
@@ -63,25 +63,45 @@ describe('transactionBreakdown routes', () => {
         .post('/api/transactionBreakdown')
         .send(transactionBreakdownSeed.transactionBreakdownMissingTax);
       expect(res.status).toBe(400);
-      expect(res.text).toBe('Missing content in payload');
 
       res = await request
         .post('/api/transactionBreakdown')
         .send(transactionBreakdownSeed.transactionBreakdownMissingTip);
       expect(res.status).toBe(400);
-      expect(res.text).toBe('Missing content in payload');
 
       res = await request
         .post('/api/transactionBreakdown')
         .send(transactionBreakdownSeed.transactionBreakdownMissingParty);
       expect(res.status).toBe(400);
-      expect(res.text).toBe('Missing content in payload');
 
       res = await request
         .post('/api/transactionBreakdown')
         .send(transactionBreakdownSeed.transactionBreakdownMissingItems);
       expect(res.status).toBe(400);
-      expect(res.text).toBe('Missing content in payload');
+    });
+
+    it('should throw an error if negative value is sent for tax, tip or item costs', async () => {
+      let res = await request
+        .post('/api/transactionBreakdown')
+        .send(transactionBreakdownSeed.transactionBreakdownNegativeTax);
+      expect(res.status).toBe(400);
```

Comment:
Let's also check for the error messages in these unit tests to make sure the request is failing for the error we are testing for.